src/App.jsx

```diff
@@ -1,37 +1,53 @@
-import "./App.css";
+import './App.css';
 
 // Components Import
-import { Navbar } from "./components/Navbar/Navbar";
-import { Footer } from "./components/Footer/Footer";
-import { SignIn } from "./components/Authentication/signin";
-import { SignUp } from "./components/Authentication/signup";
-import { Routes, Route } from "react-router-dom"
+import { Navbar } from './components/Navbar/Navbar';
+import { Footer } from './components/Footer/Footer';
+import { SignIn } from './components/Authentication/signin';
+import { SignUp } from './components/Authentication/signup';
+import { Routes, Route } from 'react-router-dom';
 
 //Pages Import
-import { Home } from "./pages/home/home";
-import { ProductListing } from "./pages/product-listing/product-listing"
+import { Home } from './pages/home/home';
+import { ProductListing } from './pages/product-listing/product-listing';
+import { Cart } from './pages/cart/cart';
+import { Wishlist } from './pages/wishlist/wishlist';
 
 //Mockman
-import Mockman from "mockman-js"; 
+import Mockman from 'mockman-js';
+import { ProtectedRoutes } from './routes/ProtectedRoutes';
 
 function App() {
-  return (
-    <div className="App">
-      {/* <Navbar />
-      <Hero />
-      <Showcase />
-      <Footer /> */}
-      <Navbar />
-      <Routes>
-        <Route path="mockman" element={ <Mockman /> } />
-        <Route path="/" element={ <Home /> } />
-        <Route path="products" element={ <ProductListing /> } />
-        <Route path="login" element={ <SignIn /> } />
-        <Route path="signup" element={ <SignUp /> } />
-      </Routes>
-      <Footer />
-    </div>
-  );
+    return (
+        <div className="App">
+            <div className="section-one">
+                <Navbar />
+            </div>
+
+            <div className="section-two">
+                <Routes>
+                    <Route path="mockman" element={<Mockman />} />
+                    <Route path="/" element={<Home />} />
+                    <Route path="products" element={<ProductListing />} />
+                    <Route path="login" element={<SignIn />} />
+                    <Route path="signin" element={<SignIn />} />
+                    <Route path="signup" element={<SignUp />} />
```

Comment:
Suggestion: You can make a separate component for Routes, this will make your App.js clean